app/config/config.yml

```diff
@@ -2,6 +2,7 @@ imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
     - { resource: services.yml }
+    - { resource: "@NewsModuleBundle/Resources/config/services.yml" }
```

Comment:
To na pewno powinno być tutaj?
Raczej lepiej stworzyć plik ``Sowp/BudgetBundle/DependencyInjection/SowpBudgetExtension.php``` 
i wpisać
````
class SowpBudgetExtension extends Extension
{
    /**
     * {@inheritdoc}
     */
    public function load(array $configs, ContainerBuilder $container)
    {
        $configuration = new Configuration();
        $config = $this->processConfiguration($configuration, $configs);

        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
        $loader->load('services.yml');
    }
}
````